data.sql

```diff
@@ -1,52 +1,25 @@
 /* Populate database with sample data. */
-INSERT INTO animals (
-        name,
-        date_of_birth,
-        escape_attempts,
-        neutered,
-        weight_kg
-    )
-VALUES ('Agumon', '2020-02-03', '0', 'true', '10.23');
-INSERT INTO animals (
-        name,
-        date_of_birth,
-        escape_attempts,
-        neutered,
-        weight_kg
-    )
-VALUES ('Gabumon', '2018-11-15', '2', 'true', '8');
-INSERT INTO animals (
-        name,
-        date_of_birth,
-        escape_attempts,
-        neutered,
-        weight_kg
-    )
-VALUES ('Pikachu', '2021-01-07', '1', 'false', '15.04');
-INSERT INTO animals (
-        name,
-        date_of_birth,
-        escape_attempts,
-        neutered,
-        weight_kg
-    )
-VALUES ('Devimon', '2017-05-12', '5', 'true', '11');
-
-
-INSERT INTO animals (
-        name,
-       date_of_birth,
-         escape_attempts,
-        neutered,
-        weight_kg
-    )
-VALUES ('Charmander', '2020-02-08', 0, false, -11),
-         ('Plantmon', '2021-11-15', 2, true, -5.7),
-         ('Squirtle', '1993-04-02', 3, false, -12.13),
-         ('Angemon', '2005-06-12', 1, true, -45),
-         ('Boarmon', '2005-06-07', 7, true, 20.4),
-         ('Blossom', '1998-10-13', 3, true, 17),
-         ('Ditto', '2022-05-14', 4, true, 22);
-
+begin;
+savepoint populate;
 
+INSERT INTO owners ( full_name, age) VALUES ('Sam Smith', '34');
+INSERT INTO owners ( full_name, age) VALUES ('Jennifer Orwell', '19');
+INSERT INTO owners ( full_name, age) VALUES ('Bob', '45');
+INSERT INTO owners ( full_name, age) VALUES ('Melody Pond', '77');
+INSERT INTO owners ( full_name, age) VALUES ('Dean Winchester', '14');
+INSERT INTO owners ( full_name, age) VALUES ('Jodie Whittaker', '38');
+INSERT INTO species ( name) VALUES ('Pokemon');
+INSERT INTO species ( name) VALUES ('Digimon');
```

Comment:
You are missing the SQL for the following project requirements:

![image](https://user-images.githubusercontent.com/38088818/197057309-cd9a8856-ad75-4b4d-bb91-4530430c6fc6.png)

- Kindly make sure sure that you UPDATE the `animals` table to meet the above requirements in the `data.sql` file.

Reply:
hi @KDlamini ,

the first modification is for the day 2 requirement and not day 3 since I have the information in my queries.sql file that is why it is not showing on my screenshot because I only added day 3 queries on my screenshot.
this is the info I have on my queries for day 2.

![Screenshot from 2022-10-21 14-23-04](https://user-images.githubusercontent.com/35464104/197185742-73ff98f2-e975-4bef-b169-55ab2a24f140.png)

Reply:
@KDlamini , 

the second modification is already added in my screenshot and i have the information related to the owner's name and their pet information.
here is the screenshot 
![Screenshot from 2022-10-21 14-24-59](https://user-images.githubusercontent.com/35464104/197186395-3ccfbe3c-668d-412f-937c-a281303a947b.png)